commands/compile/compile.go

```diff
@@ -106,7 +106,7 @@ func run(cmd *cobra.Command, args []string) {
 
 	// Check for ctags tool
 	loadBuiltinCtagsMetadata(pm)
-	ctags, err := getBuiltinCtagsTool(pm)
+	ctags, _ := getBuiltinCtagsTool(pm)
```

Comment:
May this function fail? Probably not but I think we should check the error anyway in case the behavior change in the future.

Reply:
If you enable the errorcheck linter probably It will reported a miss check